frameworks/js-bindings/bindings/script/jsb_cocos2d.js

```diff
@@ -1094,7 +1094,7 @@ cc.Color = function (r, g, b, a) {
     this.r = r || 0;
     this.g = g || 0;
     this.b = b || 0;
-    this.a = a || 255;
```

Comment:
Indeed, thank you for the fix. But please use `a === undefined` for checking, this is more efficient.